src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.cs

```diff
@@ -41,7 +70,7 @@ public sealed class OptionalFieldAttribute : Attribute
     {
         public int VersionAdded
         {
-            get { return default(int); }
+            get { throw null; }
```

Comment:
@nguerrera 

> There is also a theoretical reduction in reference assembly size. I recall compiler folks saying that a single 'throw null' IL body could be reused/shared across the assembly.

It seems you're right. When I compile the following code (in LINQPad):

```c#
object Main()
{
	throw null;
}

int NotMain(out int arg)
{
	throw null;
}
```

Then Show Metadata in ILDASM shows:

```
Method #1 (06000001) 
-------------------------------------------------------
  MethodName: Main (06000001)
  Flags     : [Private] [HideBySig] [ReuseSlot]  (00000081)
  RVA       : 0x00002050
  ImplFlags : [IL] [Managed]  (00000000)
  CallCnvntn: [DEFAULT]
  hasThis 
  ReturnType: Object
  No arguments.

Method #2 (06000002) 
-------------------------------------------------------
  MethodName: NotMain (06000002)
  Flags     : [Private] [HideBySig] [ReuseSlot]  (00000081)
  RVA       : 0x00002050
  ImplFlags : [IL] [Managed]  (00000000)
  CallCnvntn: [DEFAULT]
  hasThis 
  ReturnType: I4
  1 Arguments
    Argument #1:  ByRef I4
  1 Parameters
    (1) ParamToken : (08000001) Name : arg flags: [Out]  (00000002)
```

Notice how both RVAs are the same.